pkg/relayer/interface.go

```diff
@@ -176,6 +183,9 @@ type SessionTree interface {
 
 	// GetTrieSpec returns the trie spec of the SMST.
 	GetTrieSpec() smt.TrieSpec
+
+	// Stop stops the session tree and closes the KVStore.
```

Comment:
I feel like it would be helpful to articulate how this is similar to and/or different from the `Flush()` method (i.e. do they both free the same resources?). Perhaps consider updating the `Flush()` comment as well if that seems appropriate.